language/alias_spec.rb

```diff
@@ -157,4 +157,34 @@ def test_with_check(*args)
       end
     end.should raise_error(TypeError)
   end
+  
+  it "operates inside an instance_eval" do
+    m = Module.new
+    m.instance_eval do
+      def klass_method
+        :test
+      end
+          
+      alias klass_method_alias klass_method
+    end
+    
+    m.klass_method.should == :test
+    m.klass_method_alias.should == :test
+  end
```

Comment:
Ah no, of course it's on a module or class.
Maybe just duplicate the line 62 spec and adapt with `operates on the object's metaclass` with `operates on the class/module metaclass` and a module/class instance.